engine/common.ftl

```diff
@@ -1214,6 +1285,7 @@ behaviour.
                         }]]
                         [#break]
                     [#case "Output" ]
+                        [@debug message="output lookup" context={ "resourceId" : resource.Id, "value" : value } enabled=true /]
```

Comment:
Should this be left enabled?

Reply:
Just removed it. I hadn't pushed the change where I cleaned that out